deploy/charts/cert-manager/crds/certificates.yaml

```diff
@@ -525,7 +531,7 @@ spec:
               lastFailureTime:
                 type: string
                 format: date-time
-              notAfter:
+              notAfter:s
```

Comment:
This extra character probably breaks the update-crds script as it makes this YAML invalid. Best to remove it manually and regenerate 😄